core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -38,8 +45,22 @@
  */
 public final class MoreMeters {
 
+    private static final Logger logger = LoggerFactory.getLogger(MoreMeters.class);
+
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
+    private static final boolean MICROMETER_1_5;
+
+    static {
+        final Set<Method> methods = ReflectionUtils.getMethods(Builder.class, method ->
+                method != null && "serviceLevelObjectives".equals(method.getName()));
```

Comment:
nit: I'm not strong here, but could be inlined?
```java
private static final boolean MICROMETER_1_5 = !ReflectionUtils.getMethods(Builder.class, method ->
        method != null && "serviceLevelObjectives".equals(method.getName())).isEmpty();
```

Reply:
Fixed. 😉